landing-pages/site/content/en/announcements/_index.md

```diff
@@ -13,6 +13,15 @@ menu:
 
 **Note:** Follow [@ApacheAirflow](https://twitter.com/ApacheAirflow) on Twitter for the latest news and announcements!
 
+# December 2, 2022
+
+We’ve just released Apache **Airflow 2.5.0**. You can read all about [what's new in Apache Airflow 2.5.0]({{< ref "blog/airflow-2.5.0/index.md" >}}) blog post.
```

Comment:
Removing the link here to make the sentence more visible:

> You can read all about what's new in Apache Airflow 2.5.0 blog post.

Seems like that's missing an article somewhere, no?

How's this:

> You can read all about it in our what's new in Apache Airflow 2.5.0 blog post.

(Turning that into a suggestion to make it easier to incorporate)
```suggestion
We’ve just released Apache **Airflow 2.5.0**. You can read all about it in our [what's new in Apache Airflow 2.5.0]({{< ref "blog/airflow-2.5.0/index.md" >}}) blog post.
```